integrations/amazon_bedrock/README.md

```diff
@@ -16,6 +17,45 @@
 pip install amazon-bedrock-haystack
 ```
 
+## Contributing
+
+`hatch` is the best way to interact with this project, to install it:
+```sh
+pip install hatch
+```
+
+With `hatch` installed, to run all the tests:
+```
+hatch run test
+```
+> Note: integration tests will be skipped unless the env var COHERE_API_KEY is set. The api key needs to be valid
```

Comment:
Don't think we need the `COHERE_API_KEY` in this case. :P

Reply:
Yep, updating...

Reply:
Also the tests instructions. 👀